src/test/java/com/google/sps/data/DatastoreAvailabilityDaoTest.java

```diff
@@ -78,6 +80,14 @@
           (long) -1,
           true);
 
+  private final Availability availabilityFive =
+      Availability.create(
+          "user3",
+          new TimeRange(
+              Instant.parse("2020-07-07T22:30:00Z"), Instant.parse("2020-07-07T22:45:00Z")),
+          (long) -1,
```

Comment:
parameter comment, so readers can tell what this is supposed to be.

```
/*id=*/-1,
```
Do this for these two params

Reply:
Done for userId and id.

Reply:
Let me know if it's necessary for the scheduled field as well.

Reply:
Yes.

The point is that the bare boolean or int or similar are not self evident from the call site.

Reply:
Done